Shrine/app/src/main/java/com/example/android/authentication/shrine/CredentialManagerUtils.kt

```diff
@@ -17,16 +17,19 @@ package com.example.android.authentication.shrine
 
 import android.annotation.SuppressLint
 import android.content.Context
+import androidx.credentials.ClearCredentialStateRequest
```

Comment:
@neelanshsahai can we update the README to tell about RestoreCredentials functionality? @cy245 FYI

Reply:
Can we do this?